tweb/src/main/frontend/src/js/vue/components/ProjectsOverview.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<div class="col-12 contained">
+		<div class="scrolling-wrapper-flexbox">
+			<project-card v-for="project in projects" :project="project" v-bind:key="project.name"></project-card>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+    import ProjectCard from "./ProjectCard.vue";
+	import ProjectOverviewDetail from "../models/ProjectOverviewDetail"
+	import axios from 'axios';
+
+	@Component({
+		components: {
+			'project-card': ProjectCard
+		}
+	})
+    export default class ProjectsOverview extends Vue {
+
+		private projectDataURL = "https://raw.githubusercontent.com/FundRequest/platform-metadata/master/projects"
+		private projects: Array<ProjectOverviewDetail> = []
+
+		private _loadProjects(projects) {
+
+			axios.get(this.projectDataURL)
+				.then(response => {
+					let projectData = response.data;
+					for (var proj in projectData) {
+						projects.push({
+							name: projectData[proj]['title'],
+							description: projectData[proj]['small_description'],
+							overviewColor: projectData[proj]['background-color'],
+							projectLink: `https://fundrequest.io/requests?fase=open&projects=${proj}`,
```

Comment:
Don't hardcode https://fundrequest.io, since this is not valid for other environments. There is a class Locations.ts, add a function for that location over there. `https://fundrequest.io/requests` can be changed to `/requests`